crates/solver/src/settlement_ranker.rs

```diff
@@ -151,3 +152,54 @@ impl SettlementRanker {
         Ok((rated_settlements, errors))
     }
 }
+
+fn compare_solutions(lhs: &RatedSettlement, rhs: &RatedSettlement, decimals: i32) -> Ordering {
+    let precision = BigRational::from_i8(10).unwrap().pow(decimals);
+    let rounded_lhs = (lhs.objective_value() * &precision).floor();
+    let rounded_rhs = (rhs.objective_value() * &precision).floor();
```

Comment:
Wasn't sure if I should use checked_mul here but `Ratio<BigInt>` cannot really overflow can it?

Reply:
I don't think it can.